### PR TITLE
Latest jest

### PR DESCRIPTION
**Pull request by bpinto**
Depends on #405

### CODE REVIEW

**Comment by mAAdhaTTah** [comment]
Needs a rebase.

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "test": "npm run build && npm run test:fmt && npm run test:ts && npm run test:unit",
     "test:fmt": "prettier src/*.js test/*.js *.*d.ts extend-expect.js --check",
     "test:ts": "tsd",
-    "test:unit": "npx jest@^${JEST_VERSION:-25}"
+    "test:unit": "npx jest@^${JEST_VERSION:-27}"
```
Comment (mAAdhaTTah) [inline package.json:14]: Looking at this now, I think I set this up this way so I could test multiple Jest versions. Separate PR, but could you expand the matrix to include testing multiple Jest versions?

Comment (bpinto) [reply package.json:14]: Yeah, sorry I considered that when I did this change but I forgot to work on this.

Comment (bpinto) [reply package.json:14]: Done in https://github.com/kefirjs/jest-kefir/pull/407

**Review by mAAdhaTTah — APPROVED** [review]

**Pull request merged**